packages/kobotoolbox/test/index.js

```diff
@@ -42,7 +57,64 @@ describe('execute', () => {
   });
 });
 
-describe('getSubmissions', () => {
+describe('http.get', () => {
+  it('should GET with a query', async () => {
```

Comment:
Just a note: when we default the `format` query parameter, we should add an explicit test against it